### PR TITLE
creating a custom mosquitto broker container

### PR DESCRIPTION
**Pull request by hankpacefactory**

### CODE REVIEW

**Comment by hankpacefactory** [comment]
https://github.com/pacefactory/deployment-scripts/issues/42

**Review by jaredmcgrath — CHANGES_REQUESTED** [review]
Add this stuff to a new compose profile, e.g. in `compose/docker-compose.mosquitto.yml`.

Also need to think about API gateway support... if that's even possible? Reason being: some clients only allow us to use e.g. port 80, or 443. Is it possible that we could set this up with a reverse proxy, mapped to a URL like `https://<deployment_url>/api/mosquitto`?

File: compose/docker-compose.mosquitto.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    environment:
+      - "SCV2_PROFILE_PF_MOSQUITTO=true"
+      - "PF_MOSQUITTO_HOST=${PROJECT_PREFIX:-}pf_mosquitto"
+      - "PF_MOSQUITTO_PORT=1883"
+
+volumes:
+  mosquitto-data:
+
+networks:
+  external_network:
```
Comment (jaredmcgrath) [inline compose/docker-compose.mosquitto.yml:39]: No need for this
```suggestion
```

**Review by jaredmcgrath — COMMENTED** [review]
based on what i'm seeing here + in API gateway, this stuff should work.
Did you test locally? any luck?

**Comment by hankpacefactory** [comment]
> based on what i'm seeing here + in API gateway, this stuff should work. Did you test locally? any luck?

we'll do it live

**Review by jaredmcgrath — APPROVED** [review]

**Pull request merged**